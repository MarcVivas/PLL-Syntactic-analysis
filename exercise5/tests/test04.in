//Error on line 2
S : aAc | fA | Bdef

B : Ab | A | k
;
A : ilm | ml
;