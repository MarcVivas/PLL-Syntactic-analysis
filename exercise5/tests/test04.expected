Error at line 4: syntax error
A= i m 
S= a f


